gravitee-kubernetes-client/src/main/java/io/gravitee/kubernetes/client/KubernetesClient.java

```diff
@@ -35,24 +37,28 @@
     Single<SecretList> secretList(String namespace);
 
     /**
-     * Retrieve Kubernetes secret with given name
      * @param namespace
-     * @param secretName
-     * @return the {@link Secret} info if it exist
+     * @return list of {@link ConfigMapList}
      */
-    Maybe<Secret> secret(String namespace, String secretName);
+    Single<ConfigMapList> configMapList(String namespace);
 
     /**
-     * @param namespace
-     * @return list of {@link ConfigMapList}
+     * Get a Kubernetes element for the given location.
+     *
+     * @param location the location of the Kubernetes element. Example:
+     *    kube://default/configmap/gravitee-config
```

Comment:
Maybe we could not require the `kube://` at this level and let this prefix for the property resolver ?
Also, WDYT about `kubernetes://` instead ?